logging/bin/change_internal_password.sh

```diff
@@ -278,7 +278,7 @@ if [ "$success" == "true" ]; then
           log_notice " Elasticsearch Exporter pod to ensure OpenSearch metrics continue to be collected. "
           log_notice "                                                                                      "
           log_notice " This can be done by submitting the following command:                                "
-          log_notice "           kubectl -n $LOG_NS delete pod -l 'app=elasticsearch-exporter' "
+          log_notice "       kubectl -n logging delete pods -l 'app=prometheus-elasticsearch-exporter'      "
```

Comment:
- Hard-coded namespace name
- Use rollout restart here too?  In this case, the pod is deployed as part of a deployment (named v4m-es-exporter) rather than as part a daemonset, so the syntax is slightly different.